tests/unittests/evaluation_tests.py

```diff
@@ -1496,6 +1497,109 @@ def test_load_evaluation_view_select_fields(self):
             detection["eval2"]
 
 
+class Detections3DTests(unittest.TestCase):
+    def _create_detections(self, dims, loc, rot):
+        detection = fo.Detection(
+            dimensions=list(dims),
+            location=list(loc),
+            rotation=list(rot),
+        )
+        detections = fo.Detections(detections=[detection])
+        return detections
+
+    def _make_3d_detections_dataset(self):
+        group = fo.Group()
+        samples = [
+            fo.Sample(
+                filepath="image.png",
+                group=group.element("image"),
+            ),
+            fo.Sample(
+                filepath="point-cloud.pcd",
+                group=group.element("pcd"),
+            ),
+        ]
+
+        dataset = fo.Dataset()
+        dataset.add_samples(samples)
+        dataset.group_slice = "pcd"
+
+        sample = dataset.first()
+
+        # unit box at origin
+        dims = np.array([1, 1, 1])
+        loc = np.array([0, 0, 0])
+        rot = np.array([0, 0, 0])
```

Comment:
Ah none of your tests use a non-zero rotation. Best to include that so we know the methods are working as expected.